src/renderer/patches/platformClass.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0
+ * Vencord Desktop, a desktop app aiming to give you a snappier Discord Experience
+ * Copyright (c) 2023 Vendicated and Vencord contributors
+ */
+
+import { addPatch } from "./shared";
+
+addPatch({
+    patches: [
+        {
+            find: "platform-web",
+            replacement: {
+                // eslint-disable-next-line no-useless-escape
+                match: /return (?=__OVERLAY__\?""\.concat\((\i))/,
+                replace: "$1=$self.getPlatformClass(); return "
```

Comment:
i don't like that this patch assigns to the variable. If Discord ever updates their bundler and this becomes a const, it will likely lead to a crash.

i suggest something like
```js
match: /(?<=" platform-overlay"\):)\i/,
replace: "$self.getPlatformClass()"
```